File: Dataset/Leetcode/train/88/114.cpp
 class Solution {
public:
    void XXX(vector<int>& nums1, int m, vector<int>& nums2, int n) {

        for( int i = 0 ; i < n ; ++ m , ++ i ){
            
            // 将nums2逐一以插排方式插入nums1
            int k = m;
            while( k > 0 ){
                if( nums1[k-1] <= nums2[i] )
                    break;
                nums1[k] = nums1[k-1];
                -- k ;
            }
            
            nums1[k] = nums2[i];
        } 
    }
};

File: Dataset/Leetcode/train/14/114.java
 class Solution {
    public String XXX(String[] strs) {
        if(strs.length==1) return strs[0];
        StringBuilder sb=new StringBuilder();
        int minlen=1000;
        for(int i=0;i<strs.length;i++){
            minlen=Math.min(minlen,strs[i].length());
        }
        for(int j=0;j<minlen;j++){
            for(int i=1;i<strs.length;i++){
                if(strs[i].charAt(j)!=strs[i-1].charAt(j)){
                    return sb.toString();
                }
            }
            sb.append(strs[0].charAt(j));
        }
        return sb.toString();
    }
}

